scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/EHSMAttestationService.scala

```diff
@@ -59,8 +60,8 @@ class EHSMAttestationService(kmsServerIP: String, kmsServerPort: String,
     // TODO nonce
     val nonce: String = "test"
     if (quote == null) {
```

Comment:
why not `Log4Error.invalidInputError(quote != null, "Quote should be specified")`?
Log4Error.invalidInputError is as the same as `require`